src/SolutionCop.CommandLine/SolutionCop.CommandLine.csproj

```diff
@@ -9,7 +9,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>SolutionCop.CommandLine</RootNamespace>
     <AssemblyName>SolutionCop</AssemblyName>
-    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
```

Comment:
MSVS 2017 does not support v4.0 by default. For current moment we can increase framework.
I'll use new features in next pull requests